flow/project.py

```diff
@@ -944,10 +944,10 @@ def print_status(self, jobs=None, overview=True, overview_max_lines=None,
                      unroll=True, compact=False, pretty=False,
                      file=None, err=None, ignore_errors=False,
                      no_parallelize=False, template=None, profile=False,
-                     eligible_jobs_max_lines=None):
+                     eligible_jobs_max_lines=None, option='terminal'):
```

Comment:
I think the parameter name should be something like `output_format` or so, `option` is very generic.